code/components/citizen-devtools/src/NetEventLog.cpp

```diff
@@ -1,11 +1,11 @@
 #include <StdInc.h>
 
-#ifndef IS_FXSERVER
 #include <ResourceEventComponent.h>
 #include <ResourceManager.h>
 
+#ifndef IS_FXSERVER
```

Comment:
For moving this to citizen:server:gui, instead of this, the global define guard would instead be like `#if !defined(IS_FXSERVER) || defined(COMPILING_CITIZEN_SERVER_GUI)`.